docs/advanced-features/measuring-performance.md

```diff
@@ -160,13 +160,12 @@ export function reportWebVitals(metric) {
 > export function reportWebVitals({ id, name, label, value }) {
 >   // Use `window.gtag` if you initialized Google Analytics as this example:
 >   // https://github.com/vercel/next.js/blob/canary/examples/with-google-analytics/pages/_document.js
->   window.gtag('send', 'event', {
->     eventCategory:
+>   window.gtag('event', name, {
+>     event_category:
```

Comment:
I also see that reflected [here](https://github.com/vercel/next.js/blob/canary/examples/with-google-analytics/lib/gtag.js). Good catch!